R/quanteda_options.R

```diff
@@ -129,13 +129,13 @@ set_option_value <- function(key, value) {
             warning("Setting threads instead to maximum available ", thread["max"], call. = FALSE)
             value <- thread["max"]
         }
-        if (is.na(thread["tbb"])) {
+        # if (is.na(thread["tbb"])) {
             if (value != thread["max"])
```

Comment:
And here, we don't set it if the user wants to use the max threads? But we don't issue any message?